### ISSUE TITLE
Crashing on 1.19.4

### ISSUE THREAD

**Issue by Boy0000** [AHilyard/Iceberg#22]
```
Reported exception thrown!
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:946)
	at net.minecraft.class_310.method_1523(class_310.java:1218)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: LVT in net/minecraft/class_437::method_32633(Lnet/minecraft/class_4587;Ljava/util/List;IILnet/minecraft/class_8000;)V has incompatible changes at opcode 341 in callback iceberg.mixins.json:ScreenMixin from mod iceberg->@Inject::renderTooltipInternal(Lnet/minecraft/class_4587;Ljava/util/List;IILnet/minecraft/class_8000;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;IIII)V.
 Expected: [I, I, I, I]
    Found: []
Available: []
	at net.minecraft.class_437.handler$bmn000$iceberg$renderTooltipInternal$missing(class_437.java)
	at net.minecraft.class_437.method_32633(class_437.java:330)
	at net.minecraft.class_437.method_47941(class_437.java:265)
	at net.minecraft.class_437.method_47413(class_437.java:123)
	at net.minecraft.class_757.method_3192(class_757.java:937)
	... 6 more
```

**Comment by TotoroDude** [comment]
Same as well!!

**Comment by Blamudia** [comment]
https://gist.github.com/ChampionAsh5357/163a75e87599d19ee6b4b879821953e8
1.19.4 has lots of broken changes, mainly with render states

Some examples:
https://gist.github.com/ChampionAsh5357/163a75e87599d19ee6b4b879821953e8#guicomponent < blitOffsets removed

https://gist.github.com/ChampionAsh5357/163a75e87599d19ee6b4b879821953e8#new-registries < TransformType removed
now uses net.minecraft.world.item.ItemDisplayContext

**Comment by AHilyard** [comment]
I have now released Iceberg versions for 1.19.4, so this should no longer occur.
